src/test/java/attendance/AttendancePenaltyTest.java

```diff
@@ -0,0 +1,38 @@
+package attendance;
+
+import attendance.domain.AttendancePenalty;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.assertj.core.api.AssertionsForInterfaceTypes.assertThat;
+
+class AttendancePenaltyTest {
```

Comment:
요건 개인 취향이라 선택적인 코멘트 입니다~
`@DisplayNameGeneration` 애너테이션을 사용하면 테스트 이름에 언더바를 제거하고 출력해줍니다~

```suggestion
@DisplayNameGeneration(DisplayNameGenerator.ReplaceUnderscores.class)
class AttendancePenaltyTest {
```

![image](https://github.com/user-attachments/assets/f8951d70-2a41-4b06-bea3-e7cf3f018242)